lib/main.js

```diff
@@ -51,7 +51,7 @@ Lambda.prototype.run = function (program) {
 Lambda.prototype._runHandler = function (handler, event) {
   var context = {
     succeed: function (result) {
-      console.log('succeed: ' + JSON.stringify(result));
+      console.log('succeed: ' + result);
```

Comment:
May I ask why you remove the `JSON.stringify` here? I'm assuming this was added so the logging would dump all data instead of to a certain nested limit.